### DISCUSSION TITLE
IKEA E1743 issues

### DISCUSSION THREAD

**Discussion by bipsendk** [Koenkk/zigbee2mqtt#23051] [Devices]
Just updated to Z2M 1.38 in Home Assistant (after a restore from backup, as I had to migrate the virtual machine).

Now my E1743 buttons are marked as offline... I try to re-pair one of them. but keep getting errors trying to do so:

```
 2024-06-14 12:34:15Interview failed for '0xccccccfffebe99e1 with error 'Error: Interview failed because can not get active endpoints ('0xccccccfffebe99e1')'
Error 2024-06-14 12:34:15Failed to interview '0xccccccfffebe99e1', device has not successfully been paired

```

Any hints on what to do ? At the moment it seems impossible to get an E1743 button paired...

**Comment by rfbeiboer** [comment]
I have the same issue with this switch. I got it after removing the device from Z2M and then trying to pair it again. It keeps failing with this message

**Reply by rfbeiboer** [reply]
> I have the same issue with this switch. I got it after removing the device from Z2M and then trying to pair it again. It keeps failing with this message

I have soved it by starting an interview process while pressing the buttons once per second or so

**Comment by mquade** [comment]
Any Updates on that issue?
I tried to add two E1743 which are laying around since over a year.
One worked flawlessly and one quits every time with the massage:
```
Error 2025-03-15 19:44:56zh:controller: Interview failed for '0x943469fffe3ddc01 with error 'Error: Interview failed because can not get active endpoints ('0x943469fffe3ddc01')'
Error 2025-03-15 19:44:56z2m: Failed to interview '0x943469fffe3ddc01', device has not successfully been paired
```
Any hints?

Thanks
